snuba/cli/devserver.py

```diff
@@ -74,6 +74,33 @@ def devserver(*, bootstrap: bool, workers: bool) -> None:
                 "--storage=events",
             ],
         ),
+        (
+            "subscriptions-consumer-events",
```

Comment:
Please add the consumer group parameter to both:
`--consumer-group`
It needs to be different between the two and different from the ones used by the events consumer and transactions consumer, since the 4 consume the same topic, so, in order to consume the same messages, they have to use different consumer groups.

This is also missing the `--commit-log-group` parameter which informs the subscription consumer about which commits to look for on the commit log. Specifically `transactions_group` for the transactions subscriptions since the transactions consumer consumes on that consumer group (see above). And `snuba-consumers` for the events subscriptions consumer since it is not set and that is the default.

Also does this actually work? It seems it is missing the commit log topic `--commit-log-topic` which is `snuba-commit-log` for both. Since this topic is the one the subscription consumer follows to know when events and transactions have been successfully committed.

Reply:
Have been messing around with this a lot today and have finally got it working. Tested this using `snuba devserver` and `sentry` and saw alerts fire, so it all looks good now. Thanks for the advice!